### PR TITLE
Modals  pattern update - review required

### PR DESCRIPTION
**Pull request by chri2547**
@araiford, modals is ready for final approval!

image

### CODE REVIEW

File: _component_design/modal.md [hunk 1]
```diff
@@ -16,27 +16,27 @@ helix-ui-javascript: false
...
+## When to use
+Use modals when you want to **focus the attention of the user** by obstructing the view of all items not related to the action. Modals are useful when:
 
-- Use modals sparingly and only in situations where the user must confirm before continuing, or when the cost of an error is very high.
-- It's best to trigger a modal from a button, link, or other clickable item on the page.
-- Avoid opening modals unexpectedly.
-- Allow a user to close or cancel a modal by clicking outside of it.
-- The confirmation button **must always** reflect the action being confirmed, and follow the **Verb Noun** labeling convention. For example, if the modal title is **Permanently delete this server?**, label the confirmation button as **Delete Server**.
+- The user **action is irreversible**, such as deleting a server or resetting an API key
+- You want the user to **enter required information** before continuing to work in an application
```
Comment (nimbinatus) [inline _component_design/modal.md]: ditto

File: _component_design/modal.md [hunk 2]
```diff
@@ -46,23 +46,41 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
```
Comment (nimbinatus) [inline _component_design/modal.md]: This probably should span across the whole page as it does not have an image next to it. Remove lines 47-51 to do so.

File: _component_design/modal.md [hunk 3]
```diff
@@ -46,23 +46,41 @@ helix-ui-javascript: false
...
+
 ## Specifications
+Use the following specifications when you construct modals.
```
Comment (nimbinatus) [inline _component_design/modal.md]: Add a blank line after headings.

File: _component_design/modal.md [hunk 4]
```diff
@@ -46,23 +46,41 @@ helix-ui-javascript: false
...
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Spacing
+### Composition
+A modal contains the following:
```
Comment (nimbinatus) [inline _component_design/modal.md]: Blank line needed after heading

Comment (nimbinatus) [inline _component_design/modal.md]: Also, s/`the following`/`the following elements` (see the last row of the table at http://rackerlabs.github.io/docs-rackspace/style-guide/style/lists.html?highlight=following+adjective#introductory-text)

File: _component_design/modal.md [hunk 5]
```diff
@@ -101,9 +119,37 @@ Center modals vertically and horizontally in the browser window.
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Loading
+### Spacing
+
+Use the following spacing guidelines when you construct a modal.
+
+- Center-align modals vertically and horizontally
```
Comment (nimbinatus) [inline _component_design/modal.md]: As a sentence, this line needs a period at the end. However, it isn't parallel with the rest of the list. Suggestion:
```
- Alignment: Centered vertically and horizontally
```

File: _component_design/modal.md [hunk 6]
```diff
@@ -127,23 +173,21 @@ Refer to the [Loading pattern]({{site.url}}/components/loading-indicators.html)
...
-### Use Case: Confirmation
-
-Confirmation modals interrupt the user before they complete a given workflow and require a single decision from the user.
+## Confirmation modals
 
-The modal is triggered by the user submitting information or initiating a process that will have significant or destructive impact.
+A confirmation modal interrupts a user and requires a decision before the user can continue working. The modal is triggered when the user initiates a process or submits information that can have a significant or destructive impact.
 
-**Confirmation scenarios include the following actions:**
+**Confirmation scenarios include actions that result in:**
```
Comment (nimbinatus) [inline _component_design/modal.md]: Per style (http://rackerlabs.github.io/docs-rackspace/style-guide/style/lists.html#introductory-text), an introduction for a list should be a sentence versus a fragment.

File: _component_design/modal.md [hunk 7]
```diff
@@ -127,23 +173,21 @@ Refer to the [Loading pattern]({{site.url}}/components/loading-indicators.html)
...
 
-- Action that triggers deletion or overwriting of data
+- Deleting or overwriting data
 - Loss of unsaved data
-- Changes that will impact billing
+- Changes that impact billing
 
-**All confirmation modals include the following elements:**
+**Confirmation modals include:**
```
Comment (nimbinatus) [inline _component_design/modal.md]: Per style (http://rackerlabs.github.io/docs-rackspace/style-guide/style/lists.html#introductory-text), an introduction for a list should be a sentence versus a fragment.

**Comment by cyrichardson** [comment]
On it, @chri2547.

**Comment by cyrichardson** [comment]
@chri2547 Suggested edits here: 
[Helix_Modals_cyrEdits.docx](https://github.com/rackerlabs/design-system/files/2005855/Helix_Modals_cyrEdits.docx)

**Pull request merged**